tests/login.spec.ts

```diff
@@ -2,27 +2,27 @@ import { expect, test } from '@playwright/test';
 import { LoginPage } from '../POMs/loginPage';
 import { LoginEnvironments } from '../environments/loginEnvironments';
 
-test('Login with correct credentials', async({ page }) => {
+test('Login with correct credentials', async ({ page }) => {
     const loginPage = new LoginPage(page);
     const loginEnvironments = new LoginEnvironments(page);
 
     await page.goto(loginEnvironments.baseUrl);
     await loginPage.login(loginEnvironments.validEmail, loginEnvironments.validPassword);
-    await page.waitForURL('https://cobe-accounting.herokuapp.com/home/Offers?page=1');
+    await loginPage.assertHomePageLink();
 
 });
 
-test('Login with incorrect credentials', async({ page }) => {
+test('Login with incorrect credentials', async ({ page }) => {
     const loginPage = new LoginPage(page);
     const loginEnvironments = new LoginEnvironments(page);
 
     await page.goto(loginEnvironments.baseUrl);
     await loginPage.login(loginEnvironments.invalidEmail, loginEnvironments.invalidPassword);
-    await expect(page.locator('div').getByText(/Incorrect email or password/).first()).toBeVisible();
+    await loginPage.assertErrorMessageIsVisible();
 
 });
 
```

Comment:
Comment for the test "Show or hide password button" - how are you sure that password is displayed after the login?
First is login, and then the password should be displayed
Also, clicking show/hide button could be a function